client/src/App.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from "react";
-
 import { BrowserRouter as Router } from "react-router-dom";
 
+import "antd/dist/antd.css";
```

Comment:
whats this?

Reply:
This is for the ant design library to use it's ready components 
https://ant.design/docs/react/introduce

Reply:
if we had longer this seems unnecessary, adding a whole UI library for using it in a small amount of places makes the application bundle bigger in size